src/cup1d/likelihood/emcee_sampler.py

```diff
@@ -77,876 +120,1052 @@ def __init__(self,like=None,
-        free_param_limits=[]
+        saveDict["prior_Gauss_rms"] = self.like.prior_Gauss_rms
+        saveDict["cosmo_fid_label"] = self.like.cosmo_fid_label
+        saveDict["emu_cov_factor"] = self.like.emu_cov_factor
+        free_params_save = []
+        free_param_limits = []
         for par in self.like.free_params:
-            free_params_save.append([par.name,par.min_value,par.max_value])
-            free_param_limits.append([par.min_value,par.max_value])
-        saveDict["free_params"]=free_params_save
-        saveDict["free_param_limits"]=free_param_limits
+            free_params_save.append([par.name, par.min_value, par.max_value])
+            free_param_limits.append([par.min_value, par.max_value])
+        saveDict["free_params"] = free_params_save
+        saveDict["free_param_limits"] = free_param_limits
 
         # Sampler stuff
-        saveDict["burn_in"]=self.burnin_nsteps
-        saveDict["nwalkers"]=self.nwalkers
-        saveDict["autocorr"]=self.autocorr.tolist()
+        saveDict["burn_in"] = self.burnin_nsteps
+        saveDict["nwalkers"] = self.nwalkers
+        saveDict["autocorr"] = self.autocorr.tolist()
 
         # Save dictionary to json file in the appropriate directory
         if self.save_directory is None:
             self._setup_chain_folder()
-        with open(self.save_directory+"/config.json", "w") as json_file:
-            json.dump(saveDict,json_file)
+        with open(self.save_directory + "/config.json", "w") as json_file:
+            json.dump(saveDict, json_file)
 
         # save config info in plain text as well
         self._write_dict_to_text(saveDict)
 
         # Save plots (might have issues in some clusters)
-        try:
-            self.plot_best_fit(residuals=residuals,
-                    delta_lnprob_cut=plot_delta_lnprob_cut)
-        except:
-            print("Can't plot best fit")
-        try:
-            self.plot_prediction(residuals=residuals)
-        except:
-            print("Can't plot prediction")
-        try:
-            self.plot_autocorrelation_time()
-        except:
-            print("Can't plot autocorrelation time")
-        try:
-            self.plot_corner(usetex=(not plot_nersc),serif=(not plot_nersc),
-                    delta_lnprob_cut=plot_delta_lnprob_cut)
-        except:
-            print("Can't plot corner")
+        if plots:
+            try:
+                self.plot_best_fit(
+                    residuals=residuals, delta_lnprob_cut=plot_delta_lnprob_cut
+                )
+            except:
+                print("Can't plot best fit")
+            try:
+                self.plot_prediction(residuals=residuals)
+            except:
+                print("Can't plot prediction")
+            try:
+                self.plot_autocorrelation_time()
+            except:
+                print("Can't plot autocorrelation time")
+
+        _ = self.plot_corner(only_cosmo=True)
+        summary = self.plot_corner()
```

Comment:
Save some files below that are quite useful for plotting the results of the chains quickly